samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/jsf/validator/FinQuestionValidator.java

```diff
@@ -79,7 +79,7 @@ public void validate(FacesContext context, UIComponent component, Object value)
 		        BigDecimal rango1 = new BigDecimal(number1);
 		        BigDecimal rango2 = new BigDecimal(number2);
 		        if (rango1.compareTo(rango2) != -1) {
-		        	String error=(String)ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.DeliveryMessages", "fin_invalid_characters_error");
+		        	String error=(String)ContextUtil.getLocalizedString("org.sakaiproject.tool.assessment.bundle.DeliveryMessages", "fin_first_greater_than_second_error");
```

Comment:
I'm going to make you practice rebasing now ...

Could you add a bit of spacing here? Just String error = (String) ContextUtil .....

Reply:
@adrianfish Add the space to it!